tachyon/math/finite_fields/prime_field_unittest.cc

```diff
@@ -148,7 +148,7 @@ TYPED_TEST(PrimeFieldTest, MultiplicativeOperators) {
     F tmp = test.a;
     tmp *= test.b;
     EXPECT_EQ(tmp, test.mul);
-    tmp /= test.b;
+    ASSERT_TRUE(tmp /= test.b);
```

Comment:
I'm not sure if we can always assure the quotient to have value.

Reply:
`test.b` is guaranteed to be a concrete value (either `F(2)` or `F(3)`)